doc/js_tutorials/js_assets/js_image_classification_with_camera.html

```diff
@@ -0,0 +1,409 @@
+    <p>5.Fetch model from network and save to emscripten file system.</p>
+    <textarea class="code" rows="35" cols="100" id="codeEditor4" spellcheck="false"></textarea>
+    <p>6.The post-processing, including softmax if needed and get the top classes from the output vector.</p>
+    <textarea class="code" rows="35" cols="100" id="codeEditor5" spellcheck="false"></textarea>
+</div>
+
+<div id="appendix">
+    <h2>Model Info:</h2>
+</div>
+
+<script src="utils.js" type="text/javascript"></script>
+
+<script id="codeSnippet" type="text/code-snippet">
+inputSize = [224,224];
+mean = [104, 117, 123];
+std = 1;
+rgb = true;
+
+// record if need softmax function for post-processing
+needSoftmax = false; 
+
+// url for model, config and label files, can from native or Internet
+// for model type like onnx which doesn't need config file, just let configUrl = ""
+labelsUrl = "labels1000.txt";
+modelUrl = "bvlc_googlenet.caffemodel";
+configUrl = "bvlc_googlenet.prototxt";
+</script>
+
+<script id="codeSnippet1" type="text/code-snippet">
+let frame = new cv.Mat(video.height, video.width, cv.CV_8UC4);
+let cap = new cv.VideoCapture(video);
+
+main = async function(frame) {
+    const url = {
+        modelUrl: modelUrl,
+        configUrl: configUrl
+    }; 
+    const path = await loadModel(url);
+    const labels = await loadLables(labelsUrl);
+    const input = getBlodFromImage(frame);
+    let net;
+    if(path.configPath == "") {
+        net = cv.readNet(path.modelPath);
+    } else {
+        net = cv.readNet(path.configPath, path.modelPath);
+    }
+    net.setInput(input);
+    const start = performance.now();
+    const result = net.forward();
+    const time  = performance.now()-start;
+    const probs = softmax(result);
+    const classes = getTopClasses(probs, labels);
+
+    updateResult(classes, time);
+    setTimeout(processVideo, 0);
+    input.delete();
+    net.delete();
+    result.delete();
+}
+
+function processVideo() {
+    try {
+        if (!streaming) {
+            return;
+        }
+        cap.read(frame);
+        main(frame);
+    } catch (err) {
+        utils.printError(err);
+    }
+}
+
+setTimeout(processVideo, 0);
+</script>
+
+<script id="codeSnippet2" type="text/code-snippet">
+loadLables = async function(labelsUrl){
+    document.getElementById('status').innerHTML = '';
+    let response = await fetch(labelsUrl);
+    let label = await response.text();
+    label = label.split('\n');
+    document.getElementById('status').innerHTML = '';
+    return label;
+}
+</script>
+
+<script id="codeSnippet3" type="text/code-snippet">
+getBlodFromImage = function(frame) {
+    let matC3 = new cv.Mat(frame.matSize[0],frame.matSize[1],cv.CV_8UC3);
+    cv.cvtColor(frame, matC3, cv.COLOR_RGBA2RGB);
+    let input = cv.blobFromImage(matC3, std, new cv.Size(inputSize[0], inputSize[1]), 
+                                 new cv.Scalar(mean[0], mean[1], mean[2]), rgb);
+    matC3.delete();
+    return input;
+}
```

Comment:
Please avoid code duplication - create a separate .js script with common methods

Reply:
Because my original idea was to show the processing function into each example webpage to users, so I put the similar code in each html file.

Reply:
Not a good idea as for me. If one of the code snippets will have a bug - it's hard to find all the copies. So I'd like to suggest to unite the logic.

Reply:
I believe its possible to use `utils.loadCode` in case of shared `.js` file as well